### ISSUE TITLE
hashivault_approle_role_create does not support secret_id_ttl, period, token_ttl, token_max_ttl

### ISSUE THREAD

**Issue by vuminhkh** [TerryHowe/ansible-modules-hashivault#63]
Hello,

Eveything is in the title, for now i'm using a workaround by using the generic end point:

```
hashivault_write:
    token: "{{ vault_admin_token.stdout }}"
    secret: '/auth/approle/role/goldfish'
    data:
      role_name: "goldfish"
      policies: "default,goldfish"
      secret_id_num_uses: 1
      secret_id_ttl: "5m"
      period: "24h"
      token_ttl: 0
      token_max_ttl: 0
```
Thank you in advance for your help.

Regards,

Khang

**Comment by TerryHowe** [comment]
Not released, let me know what you think:

https://github.com/TerryHowe/ansible-modules-hashivault/commit/71ebf3fc01bcccf36e89b7800062ea87e7a51cf4

**Comment by vuminhkh** [comment]
Thanks, I'm not really python developer, but I'll be happy to test once it's released, or if you tell me how to test the snapshot version.

**Comment by TerryHowe** [comment]
Okay, released 3.9.5.  Any problems, just open another issue.
